docs/_articles/en/lwc/lwc-get-started.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Quick Start LWC Development
+lang: en
+---
+
+## Overview
+You’ll create an LWC component called `contactList` that displays contact names and phone numbers in your App. We’ll get the contact information to display from an Apex class called `ContactController`.
+
+Before you start, install [Salesforce Extensions for Desktop](./en/vscode-desktop/install) or [Set Up Code Builder](./en/codebuilder/cb-setup)
+
+### Add an Apex Class that Queries Contacts
+1. In VS Code, run the command **SFDX: Create Project** to create a Salesforce DX project if you don’t have one.
+2. Log in to the org to which you want to add the LWC.
```

Comment:
Would it be better if giving the instruction of command like 
"Run "SFDX: Create a Default Scratch Org" or "SFDX: Authorize an Org" to set one."?

Reply:
Great callout! 🙌🏼